src/runtime/runtime.cs

```diff
@@ -43,39 +43,24 @@ public class Runtime
 #error You must define either UCS2 or UCS4!
 #endif
 
-        // C# compiler copies constants to the assemblies that references this library.
-        // We needs to replace all public constants to static readonly fields to allow
-        // binary substitution of different Python.Runtime.dll builds in a target application.
-
-        public static string pyversion => _pyversion;
```

Comment:
@filmor I think a unit test depends on these public properties

Reply:
Which one? I was used in a few places, but only to check for Python 2 vs 3.

Reply:
@filmor you already removed them actually from TestPythonEngineProperties.cs
Seems like this might require an entry in the changelog since its a breaking change technically

Reply:
Yeah, I could do that, but `pyversion` etc. where never really consumables in the first place. I'd like to move the whole `Runtime` to `internal` as a single Changelog entry, no need to do this for every single component.

Reply:
@filmor moving `Runtime` to `internal` deserves its own tracking issue.

Reply:
Agreed: https://github.com/pythonnet/pythonnet/issues/1167